build-system/server/new-server/transforms/utilities/cdn.ts

```diff
@@ -97,19 +111,16 @@ export function CDNURLToRTVURL(
   pathnames: [string | null, string | null] = [null, null],
   extension: string = '.js'
 ): URL {
-
   const [overwriteablePathname, newPathname] = pathnames;
   if (url.pathname === overwriteablePathname && newPathname !== null) {
     url.pathname = newPathname;
   }
 
-  const parsedPath = parse(`/rtv/${mode}/${url.pathname}`);
+  const parsedPath = parse(`/rtv/${mode}/${url.pathname.replace(/^\//, '')}`);
```

Comment:
Added this replacement since we'd end up with double slashes:

```
https://cdn.ampproject.org/rtv/123456789012345//v0.js
```

But if `pathname` is guaranteed to contain the leading slash, we can remove the added slash after `mode`.

Reply:
yep